### ISSUE TITLE
Feature request: keyboard navigation support in expansion item

### ISSUE THREAD

**Issue by kdmon** [quasarframework/quasar#3758]
### Summary
Expansion item does not expand/collapse when pressing enter key when the expansion item is in focus.

### What did you get as the error?
No error message.

### What were you expecting?
Expansion content should be expanded (or collapsed if already expanded) when expansion item is in focus and enter key is pressed.

### What steps did you take, to get the error?
Create page with expansion item.
Set tab focus to expansion item.
Press enter to expand/collapse item.
Nothing happens.

### Consequences
Keyboard navigation is critical for WCAG/accessibility compliance. This is a major obstacles for products with strict WCAG compliance needs (e.g. many public sector projects).

### Software version
OS: all desktop browsers presumably (tested in Chrome)
Quasar: v.1.0+

### Reproduction codepen
Codepen: https://codepen.io/anon/pen/vMYGRy

**Comment by rstoenescu** [comment]
Hi,

This was broken recently and it's already fixed in future beta.14.

**Comment by rstoenescu** [comment]
Hi,

We take good care of keyboard accessibility, however it seems like this got broken recently, so thank you for reporting.

Fix will be available in beta.14.
